engine/common/mod_bmodel.c

```diff
@@ -2879,6 +2897,26 @@ static int Mod_LumpLooksLikeEntities( const char *lump, const size_t lumplen )
 	return Q_memmem( lump, lumplen, "\"classname\"", sizeof( "\"classname\"" ) - 1 ) != NULL ? 1 : 0;
 }
 
+/*
+=================
+Mod_LoadBSPHeader
+
+=================
+*/
+_inline dheader_t* Mod_LoadBSPHeader( const byte *mod_base )
+{
+	dheader_t *header = (dheader_t *)mod_base;
+#ifdef XASH_BIG_ENDIAN
+	LittleLongSW( header->version );
```

Comment:
You're potentially writing to `const` here. While this memory isn't read-only, I'm not sure if it's a good solution here. I think we can remove `const` modifier from BSP model loader for now.

Also, you can use `static inline` instead of `_inline`. Or even just `static`, since most compilers do ignore `inline` keyword.

`_inline` is a legacy macro definition I forgot about, it's almost not used in new code.